### ISSUE TITLE
public cow runs - what if they go for the crowned one

### ISSUE THREAD

**Issue by loldermond** [kolton/d2bot-with-kolbot#2167]
are there ways to chicken out as soon as the cowking is being approached?

**Comment by Fa-b** [comment]
I think the times of you not beeing able to open cowgames anymore after the king was killed are long over... no need to worry about that.

**Comment by loldermond** [comment]
i did not know that. thanks oO

**Comment by Fa-b** [comment]
No, no reliable source for it. And after I googled trying to find one I'm not sure anymore if my statement is correct at all! Sorry for maybe misleading information.
What appears to be the case is: if the person killing cowking did not complete that difficulty (does not have baal quest finished), there will be no effect.

**Comment by loldermond** [comment]
so is there a way to safely run cows on bot public? everyone is asked

**Comment by InterestingIndeed** [comment]
> so is there a way to safely run cows on bot public? everyone is asked

just do it like the bots do it:

1. Have a level 1 char open the game that doesnt party anyone.
2. Use your "real" char to open cows and do the cows, dont party the level 1 char.
3. Profit.

**Comment by InterestingIndeed** [comment]
> No, no reliable source for it. And after I googled trying to find one I'm not sure anymore if my statement is correct at all! Sorry for maybe misleading information.
> What appears to be the case is: if the person killing cowking did not complete that difficulty (does not have baal quest finished), there will be no effect.

> Quote (CookiesssD2 @ 17 Jan 2020 14:16)
> You have to be in the cow level at the time the cow king dies to lose the ability to create the cow portal.
> 
> - So, if you go to town or leave the game BEFORE the cow king dies, whether you're in the party or not you don't get affected
> - If you're not in party BUT you're in the cow level, you are affected

This guy is a trustable source for me ;)

**Comment by Fa-b** [comment]
Yeah well I thought you wanted to run through the cow level yourself :)
